src/status_im/contexts/wallet/wallet_connect/modals/sign_message/view.cljs

```diff
@@ -12,6 +14,43 @@
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
+(def ^:constant typed-data-methods
+  #{constants/wallet-connect-eth-sign-typed-v4-method
+    constants/wallet-connect-eth-sign-typed-method})
+
+(defn typed-data-field
+  [{:keys [label value]}]
+  [quo/data-item
+   {:card?           false
+    :container-style (merge style/data-item
```

Comment:
`merge` is known to be quite slow, so if we can avoid it the better. Here you can use `assoc`, or better extract the `container-style` map into a var because it's a static map.